### PR TITLE
Correct refresh of workspace browser

### PR DESCRIPTION
**Pull request by marimano**
1) Show correct section after refresh workspace browser (if custom node section was selected before refreshing make it stay and after that); 
2) Make a loaded from a file workspace appear in workspace browser

### CODE REVIEW

File: app/scripts/models/BaseSaveUploader.js [hunk 1]
```diff
@@ -49,6 +49,11 @@ define(['backbone', 'SaveFileMessage', 'SetModelPositionMessage', 'ClearWorkspac
 
             if (ws.get('nodes').length)
                 app.trigger('ws-data-loaded');
+
```
Comment (pboyer) [inline app/scripts/models/BaseSaveUploader.js]: I see that this fixes, a problem, but I'd rather you expose this as an event on SaveUploader that App can respond to.  For example, "request-workspace-browser-refresh."

File: app/scripts/views/WorkspaceBrowserView.js [hunk 2]
```diff
@@ -1,5 +1,11 @@
 define(['backbone', 'WorkspaceBrowserElementView'], function(Backbone, WorkspaceBrowserElementView) {
 
+  var itemsToClick = {
+      project: 'project',
+      customNode: 'custom node'
+  };
+  var clicked = itemsToClick.project;
```
Comment (pboyer) [inline app/scripts/views/WorkspaceBrowserView.js]: This is a bit of an odd name, how about "lastClickedItem."

**Comment by pboyer** [comment]
Thanks @marimano!  One thing I'd like you to consider when adding new code from now on is where you can use an event instead of requiring an explicit object reference.  Generally, I'd like to start to reduce our usage of App in subcomponents and use an event model instead.  Does that make sense?  If you can fix these issues I'll merge this in.  Thanks and happy new year!

**Comment by marimano** [comment]
@pboyer, thanks for notes! I've updated the PR according to them

**Comment by pboyer** [comment]
Wonderful!  Thanks @marimano!

**Pull request merged**